test/e2e/storage/csi_mock_volume.go

```diff
@@ -1165,6 +1169,68 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			})
 		}
 	})
+
+	ginkgo.Context("CSIServiceAccountToken [Feature:CSIServiceAccountToken]", func() {
+		var (
+			err error
+		)
+		tests := []struct {
+			desc                   string
+			deployClusterRegistrar bool
```

Comment:
We don't support clusterregistrar anymore. Maybe rename this to "deployCSIDriverObject"?

Reply:
done